iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -8,40 +8,122 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public static class MessageSystemPropertyNames
     {
+        /// <summary>
+        /// A user-settable identifier for the messages. If this value is not supplied by the user,
+        /// the service client will set this to a new GUID.
+        /// </summary>
         public const string MessageId = "message-id";
 
+        /// <summary>
+        /// Lock token of the received message.
+        /// A unique identifier for a cloud-to-device message used to complete, reject or abandon the message.
+        /// This value is provided to resolve race conditions when completing, rejecting, or abandoning messages.
+        /// <remarks>
+        /// If the lock token expires, the cloud-to-device message needs to be received again to complete, reject or abandon
+        /// the message without errors.
+        /// </remarks>
+        /// </summary>
         public const string LockToken = "iothub-messagelocktoken";
 
+        /// <summary>
+        /// A number (unique per device-queue) assigned by IoT hub to each cloud-to-device message.
+        /// </summary>
         public const string SequenceNumber = "iothub-sequencenumber";
 
+        /// <summary>
+        /// A destination specified in cloud-to-device messages.
+        /// </summary>
         public const string To = "to";
 
+        /// <summary>
+        /// Number of times the message has been previously delivered.
```

Comment:
attempted to be delivered?? I'm not sure, mostly asking.

Reply:
Good question. I don't know the answer. Let me find out and add clarity.

Reply:
I have asked for clarification on this as well as the team is also not clear. Will remove the xml comments and send another PR once we get the answers.